hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/procedure/TestClusteringProcedure.scala

```diff
@@ -695,8 +695,7 @@ class TestClusteringProcedure extends HoodieSparkProcedureTestBase {
            |  type = 'cow',
            |  preCombineField = 'ts',
            |  hoodie.index.type = 'BUCKET',
-           |  hoodie.bucket.index.hash.field = 'id',
-           |  hoodie.datasource.write.recordkey.field = 'id'
```

Comment:
Note, no need this, because we add `HoodieOptionConfig.mapSqlOptionsToDataSourceWriteConfigs` for `catalogProperties`, (bug fix)